### PR TITLE
Update to RequestMaxBytes to align with updated API and regenerate fixtures

### PR DESCRIPTION
**Pull request by Integralist**
The API backend changed behaviour. This meant that the previous _max_ value of `1024` could no longer be used as it was smaller than the allowed _minimum_ of `2048`.

### CODE REVIEW

File: fastly/kafka_test.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ func TestClient_Kafkas(t *testing.T) {
 			TLSClientCert:    clientCert,
 			TLSClientKey:     clientKey,
 			ParseLogKeyvals:  true,
-			RequestMaxBytes:  1024,
+			RequestMaxBytes:  2048,
```
Comment (phamann) [inline fastly/kafka_test.go:41]: One minor nit. Seeing as this number is repeated 3 times in this file and an API enforced limit, it feels like it should be factored into a constant value at the top of the file. Just my style preference.

**Review by phamann — APPROVED** [review]
👍🏻 LGTM, other than nit about use of constant.

File: fastly/kafka_test.go (hunk 1 above)
Comment (Integralist) [reply fastly/kafka_test.go:41]: I'm usually a "more than twice, refactor" kinda guy. So I can dig it 👍🏻

**Review by doramatadora — APPROVED** [review]
I can't comment on the fixtures but the code changes LGTM 🎉

**Pull request merged**